### ISSUE TITLE
positionOffset (still) wrong in openPMD output

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#4319]
With the bug fix #4283 the discrepancy between the mesh positionOffset and the particle positionOffset was fixed. Now both should agree. 

This week @JessicaTiebel had a look at the *default* LWFA example and observed discrepancies between the mesh and the particle positions. The PIConGPU version used was `0.7.0-dev` with `commit 5788150dbba29288be24c4a886e51f279fdde1d0` from Oct 5 2022 and thus should include fix #4283. 

A detailed analysis of the evolution between mesh position (mesh's `positionOffset` and mesh's `positionOffset` + mesh's length; in y) and the maximum and minimum particle `positionOffset` show that the particle `positionOffsets` offsets from the meshes range as soon as the moving window starts. 

The following plot shows in the green the range of the mesh (`E_x`) and in gray are minimal and maximal position in cells. 
In red the difference between minimal and maximal particle position is plotted with the dashed red line marking the mesh's hight in y (const. = 768 cells). 
Obviously it is not just a simple offset error since the range of particle positions changes constantly (and not only in the beginning, when we have no particles in the first few cells)

![grafik](https://user-images.githubusercontent.com/5121158/195629618-313bb7ae-829c-4cd1-a45c-406118e9e302.png)

The python code to generate the plot is:
```python
import numpy as np
import openpmd_api as io
import matplotlib.pyplot as plt

series = io.Series("__path_to_sim__/simOutput/openPMD/simData_%T.bp",
                  io.Access.read_only)

# arrays for data
iterations_list = []
min_max_pos = np.zeros((len(series.iterations), 2)) # particles min max position
mesh_pos = np.zeros((len(series.iterations), 2)) # mesh globalOffset and mesh length (should be const.) 

# read from all iterations
for index_tmp, tmp in enumerate(series.iterations):
    iterations_list.append(tmp)
    iteration = series.iterations[tmp]
    e_h = iteration.particles['e']
    posOffset = e_h['positionOffset']['y'][:]
    series.flush()
    min_max_pos[index_tmp, :] = np.amin(posOffset), np.amax(posOffset) # min max particle position
    
    mesh_pos[index_tmp, 0] = iteration.meshes['E'].get_attribute('gridGlobalOffset')[1] # 1 = y offset
    mesh_pos[index_tmp, 1] = iteration.meshes['E']['x'].shape[1] # 1 = y mesh length
    series.flush() # not realy needed 

# plot results
plt.figure(facecolor="w")

# mesh range
plt.fill_betweenx(iterations_list, x1=mesh_pos[:, 0], x2=mesh_pos[:, 0] + mesh_pos[:, 1], color="green", alpha=0.5)

# particle min max position
plt.plot(min_max_pos[:, 0], iterations_list, color="gray")
plt.plot(min_max_pos[:, 1], iterations_list, color="gray")

# difference between min max and max mesh range
plt.plot(min_max_pos[:, 1] - min_max_pos[:, 0], iterations_list, color="red")
plt.plot(mesh_pos[:, 1], iterations_list, "--", color="red")

plt.xlim(0, 5000)
plt.ylim(0, 5000)
plt.xlabel("position [cells]", fontsize=18)
plt.xticks(fontsize=14)
plt.ylabel("time step", fontsize=18)
plt.yticks(fontsize=14)

plt.tight_layout()
plt.show()
```

**Comment by PrometheusPi** [comment]
The fix suggested in the comments of #4283 is not solving the problem. 
![grafik](https://user-images.githubusercontent.com/5121158/195868633-54000293-b631-402b-8b9a-c8b4765736d0.png)

**Comment by PrometheusPi** [comment]
The bug was fixed with #4339. 
Thanks again to @JessicaTiebel for discovering and reporting this bug.
